packages/core-components/src/components/Table/Table.tsx

```diff
@@ -373,7 +373,7 @@ export function Table<T extends object = {}>(props: TableProps<T>) {
       const newData = (data as any[]).filter(
         el =>
           !!Object.entries(selectedFilters)
-            .filter(([, value]) => !!value.length)
+            .filter(([, value]) => !!(value as []).length)
```

Comment:
Just curious but why is this change needed? I might not seeing that some of the other changes might need it

Reply:
that is because the TSC at least on my machine was failing to build due to it being `type unknown` and seeing that as an error happy to revert. 
![Screen Shot 2022-10-17 at 8 13 08 am](https://user-images.githubusercontent.com/18130488/196060813-e089dcf8-dc81-4e58-8dd9-f921ea35d789.png)

Reply:
Interesting, I only asked as I was working in that exact file for a PR I have open. I didn't run into any issues running `tsc`. Let's leave this up to the maintainers themselves